security/cmd/istio_ca/main.go

```diff
@@ -264,7 +274,14 @@ func createCA(core corev1.SecretsGetter) ca.CertificateAuthority {
 	var caOpts *ca.IstioCAOptions
 	var err error
 
-	if opts.selfSignedCA {
+	if opts.upstreamCAAddress != "" {
+		log.Info("Rely on upstream CA to provision CA certificate")
+		caOpts, err = ca.NewIntegratedIstioCAOptions(opts.upstreamCAAddress, opts.upstreamCACertFile, opts.upstreamAuth,
```

Comment:
Consider breaking line, this looks long.

Reply:
Sure. I will to this in the next PR.